client/public/assets/css/app.css

```diff
@@ -253,4 +252,7 @@ footer * {
 
 .added-padding: {
   padding-bottom: 30px;
+}
+.vertically-align {
```

Comment:
Let's rename to `.flux-logo` so that it doesn't sound like a global helper class for vertically aligning other elements. Rename in the `App.js` file too.